keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/Build.kt

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.keel.api.artifacts
+
+/**
+ * An immutable data class that represents a published software artifact in the Spinnaker ecosystem.
+ *
+ * This class mirrors [com.netflix.spinnaker.igor.build.model.GenericBuild], but without all the Jackson baggage.
+ */
+
+data class Build(
+  val building: Boolean = false,
+  val fullDisplayName: String? = null,
+  val name: String? = null,
```

Comment:
Does it makes sense to have an object of this class without at least one of these two fields present?

Reply:
what do you mean? which 2 fields?

Reply:
Sorry, the ones on the lines selected for the comment, `fullDisplayName` and `name`.